src/osdc/Objecter.cc

```diff
@@ -4526,14 +4526,15 @@ void Objecter::dump_requests(Formatter *fmt)
 
 void Objecter::_dump_ops(const OSDSession *s, Formatter *fmt)
 {
+  auto now = ceph::coarse_mono_clock::now();
   for (map<ceph_tid_t,Op*>::const_iterator p = s->ops.begin();
        p != s->ops.end();
        ++p) {
     Op *op = p->second;
     fmt->open_object_section("op");
     fmt->dump_unsigned("tid", op->tid);
     op->target.dump(fmt);
-    fmt->dump_stream("last_sent") << op->stamp;
+    fmt->dump_stream("duration") << (now - op->stamp);
```

Comment:
Why not just add a `duration` field and keep `last_sent`?

Reply:
last_sent is obscure and sometimes confusable. for example:
            "tid": 378,
            "pg": "2.2b4ef7b2",
            "osd": 2,
            "object_id": "10000000003.00000000",
            "object_locator": "@2",
            "target_object_id": "10000000003.00000000",
            "target_object_locator": "@2",
            "paused": 0,
            "used_replica": 0,
            "precalc_pgid": 0,
            "last_sent": "162711s",
            "attempts": 1,
            "snapid": "head",
            "snap_context": "1=[]",
            "mtime": "2017-12-21 10:57:02.0.514084s",
            "osd_ops": [
                "write 173056~512 [2@0]"
            ]

what does 162711s mean? (162711 seconds after the system starts)
Maybe we should output it in a more readable format